cdk/src/open-data-platform/data-plane/data-import/write-aggregate-zip-demographic-data.handler.ts

```diff
@@ -0,0 +1,90 @@
+import { RDSDataService } from 'aws-sdk';
+import { BatchExecuteStatementRequest, SqlParametersList } from 'aws-sdk/clients/rdsdataservice';
+import { geoJsonHandlerFactory } from './handler-factory';
+import { AggregateUsDemographicTableRowBuilder } from '../model/aggregate_us_demographic_table';
+
+// As of 2022-08-02, this should have 39,759 rows.
+const s3Params = {
+  Bucket: 'opendataplatformapistaticdata/demographics',
+  Key: 'zipcode_demographics.geojson',
+};
+
+const SCHEMA = 'public';
+
+/**
+ * Writes rows into the aggregate US demographics table for zip codes.
+ *
+ * @param rdsService: RDS service to connect to the db.
+ * @param rows: Rows to write to the db.
+ */
+async function insertBatch(
+  rdsService: RDSDataService,
+  rows: SqlParametersList[],
+): Promise<RDSDataService.BatchExecuteStatementResponse> {
+  const batchExecuteParams: BatchExecuteStatementRequest = {
+    database: process.env.DATABASE_NAME ?? 'postgres',
+    parameterSets: rows,
+    resourceArn: process.env.RESOURCE_ARN ?? '',
+    schema: SCHEMA,
+    secretArn: process.env.CREDENTIALS_SECRET ?? '',
+    sql: `INSERT INTO aggregate_us_demographics (census_geo_id,
+                                                 geo_type,
+                                                 name,
+                                                 median_year_built,
+                                                 median_income,
+                                                 home_age_index,
+                                                 income_index,
+                                                 weighted_national_adi,
+                                                 weighted_state_adi,
+                                                 population_count,
+                                                 geom)
+          VALUES (:census_geo_id,
+                  :geo_type,
+                  :name,
+                  :median_year_built,
+                  :median_income,
+                  :home_age_index,
+                  :income_index,
+                  :weighted_national_adi,
+                  :weighted_state_adi,
+                  :population_count,
+                  ST_AsText(ST_GeomFromGeoJSON(:geom))) ON CONFLICT (census_geo_id) DO NOTHING`,
+  };
+  return rdsService.batchExecuteStatement(batchExecuteParams).promise();
+}
+
+/**
+ * Maps a data row to a table row ready to write to the db.
+ * @param row: row with all data needed to build a [ZipcodeTableRow].
+ */
+function getTableRowFromRow(row: any): SqlParametersList {
+  const value = row.value;
+  const properties = value.properties;
+  return (
+    new AggregateUsDemographicTableRowBuilder()
+      .censusGeoId(properties.ZCTA5CE10)
+      .geoType('zip_code') // TODO make constant
```

Comment:
I think you mean a postgres type so that it's a constant in the db, for now could you make this an enum even if what you are writing to the db is a string?

Reply:
I actually meant a literal string constant and forgot to make the change / delete this comment, but this is a much better idea. Thanks!